token.go

```diff
@@ -55,6 +55,17 @@ func NewWithClaims(method SigningMethod, claims Claims) *Token {
 
 // SignedString creates and returns a complete, signed JWT.
 // The token is signed using the SigningMethod specified in the token.
+// Note it is not advised to provide a []byte which was converted from a 'human readable' string using a subset of ASCII characters.
+// i.e.
+// token.SignedString([]byte(os.Getenv("JWT_SECRET")))
+// where `JWT_SECRET“ is some random string containing the subset of ASCII characters which is synonymous with string based keys
+//
+// To maximize entropy, you should ideally be providing a []byte key which was produced from a cryptographically random source.
+// i.e. crypto/rand https://pkg.go.dev/crypto/rand#Read
+//
+// Storing keys in the environment can be done by base64 encoding the cryptographically random []byte.
+// Reading keys from the environment can be done by base64 decoding the environment variable to retrieve the original cryptographically random []byte.
+// i.e. encoding/base64 https://pkg.go.dev/encoding/base64#Encoding.DecodeString
```

Comment:
Is this a sensible doc note on `*Token.SignedString()`?
I imagine this is the main entrypoint for callers into the underlying methods `Sign` call